regulation-worker/internal/client/client.go

```diff
@@ -37,12 +37,19 @@ func (j *JobAPI) Get(ctx context.Context) (model.Job, error) {
 		return model.Job{}, err
 	}
 
+	workspaceToken := config.GetEnv("CONFIG_BACKEND_TOKEN", "22JnFdS3ZKDd0UfuEYowUeNi9fe")
+	req.SetBasicAuth(workspaceToken, "")
+	req.Header.Set("Content-Type", "application/json")
+
 	client := &http.Client{}
 	resp, err := client.Do(req)
 	if err != nil {
 		return model.Job{}, err
 	}
 	defer resp.Body.Close()
+	if err != nil {
+		fmt.Println(err)
+	}
```

Comment:
what do we error check here ?

Reply:
sorry, wanted to added `resp.Body.Close()` inside a defer func and check error, if not closed properly. Forgot to add `defer func`...added.